opentracing_shim/src/main/java/io/opentelemetry/opentracingshim/TelemetryInfo.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.opentracingshim;
+
+import io.opentelemetry.distributedcontext.DistributedContextManager;
+import io.opentelemetry.trace.Tracer;
+
+/**
+ * Utility class that holds a Tracer, a DistributedContextManager, and related objects that are core
+ * part of the OT Shim layer.
+ */
+final class TelemetryInfo {
```

Comment:
Why do we need this class?  Seems we could reference the below fields independently directly.

Reply:
I found myself passing these objects around, and wasn't too happy about it ;) A matter of style only.

Reply:
given that it's private, no strong opinion, but I'd lean towards removing.